packages/api/src/platforms/vtex/loaders/sku.ts

```diff
@@ -39,8 +39,8 @@ export const getSkuLoader = (_: Options, clients: Clients) => {
     const missingSkus = skus.filter((sku) => !sku)
 
     if (missingSkus.length > 0) {
-      throw new Error(
-        `Search API did not return the following skus: ${missingSkus.join(',')}`
+      throw new NotFoundError(
+        `Search API did not found the following skus: ${missingSkus.join(',')}`
```

Comment:
[By the time I tested it](https://github.com/vtex/faststore/pull/1220#pullrequestreview-936690157), it was not working as expected.

Maybe it's a good opportunity to improve this logic here. What do you think?

Reply:
NICE! Sure!